src/test/kotlin/com/pubnub/contract/step/KeysetStep.kt

```diff
@@ -38,4 +38,15 @@ class KeysetStep(private val world: World) {
             subscribeKey = ContractTestConfig.subKey
         }
     }
+
+    @Given("the demo keyset")
+    fun the_demo_keyset() {
+        MatcherAssert.assertThat(ContractTestConfig.pubKey, Matchers.notNullValue())
```

Comment:
What about replacing body with:
i_have_a_keyset_with_access_manager_enabled_without_secret_key()
?

Reply:
I don't know. Is it the same because it's the nature of the requirement or is it the same by accident?

Reply:
I can do whatever. I think those steps are mostly not interesting for SDK at all

Reply:
I see